test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/PatchTests.cs

```diff
@@ -95,5 +95,141 @@ public async Task GivenAServerThatSupportsIt_WhenSubmittingAForbiddenPropertyPat
 
             Assert.Equal(HttpStatusCode.BadRequest, exception.Response.StatusCode);
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenAServerThatSupportsIt_WhenSubmittingAReplacePatch_ThenServerShouldPatchCorrectly()
+        {
+            var poco = Samples.GetDefaultPatient().ToPoco<Patient>();
+            FhirResponse<Patient> response = await _client.CreateAsync(poco);
+            Assert.Equal(AdministrativeGender.Male, response.Resource.Gender);
+            Assert.NotNull(response.Resource.Address);
+
+            string patchDocument =
+                "[{\"op\":\"replace\",\"path\":\"/gender\",\"value\":\"female\"}]";
+
+            using FhirResponse<Patient> patch = await _client.PatchAsync(response.Resource, patchDocument);
+
+            Assert.Equal(HttpStatusCode.OK, patch.Response.StatusCode);
+            Assert.Equal(AdministrativeGender.Female, patch.Resource.Gender);
+        }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenAServerThatSupportsIt_WhenSubmittingAddBirthdatePatch_ThenServerShouldPatchCorrectly()
```

Comment:
Merge all "ShouldPatchCorrectly" tests as a theory. Similar to how it's done here: https://github.com/edaena/fhir-server/blob/9a43c2ec3538dc452cc393d93b51167450d661ad/test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/PatchTests.cs#L85

Reply:
Done